Makefile

```diff
@@ -23,7 +23,8 @@ typecheck:
 _buf: clean
 	rm -rf src/viam/gen
 	chmod +x plugin/main.py
-	buf generate buf.build/viamrobotics/api
+	$(eval API_VERSION := $(shell grep 'API_VERSION' src/viam/version_metadata.py | awk -F '"' '{print $$2}'))
```

Comment:
I think there's one too many `$` in here? The command as written fails for me locally, but changing to `print $2` works.

Reply:
Oh that's interesting. It ran well for me. I just tried making that change and running it. It fails when I only have 1 `$`.

Reply:
From what I understand, we need two `$` since only using one means `make` thinks that it's a makefile variable.  `make` should be passing `$$2` as `$2` to `awk`. If this is true, I wonder why it's not working for you, and how it works with only one `$`. Could it be a difference in versions?

Reply:
Interesting... Well I've been running just on the command line, if it works for you from the `makefile` then it's probably fine, thanks for following up :)

Reply:
Oh, I've been testing by running `make buf` on the command line.
If you copy/paste `grep 'API_VERSION' src/viam/version_metadata.py | awk -F '"' '{print $$2}'` into the terminal, then it will fail. Is that what you did?